app/code/Magento/Sales/Controller/Adminhtml/Order/Invoice/NewAction.php

```diff
@@ -37,22 +38,31 @@ class NewAction extends \Magento\Backend\App\Action implements HttpGetActionInte
      */
     private $invoiceService;
 
+    /**
+     * @var OrderRepositoryInterface
+     */
+    private $orderRepository;
+
     /**
      * @param Action\Context $context
      * @param Registry $registry
      * @param PageFactory $resultPageFactory
      * @param InvoiceService $invoiceService
+     * @param OrderRepositoryInterface $orderRepository
      */
     public function __construct(
         Action\Context $context,
         Registry $registry,
         PageFactory $resultPageFactory,
-        InvoiceService $invoiceService
+        InvoiceService $invoiceService,
+        OrderRepositoryInterface $orderRepository
```

Comment:
Please inject new dependencies in backward-compatible manner: https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter